docs/prod-host-setup.md

```diff
@@ -6,9 +6,7 @@ Using Jailer in a production Firecracker deployment is highly recommended,
 as it provides additional security boundaries for the microVM.
 The Jailer process applies
 [cgroup](https://www.kernel.org/doc/Documentation/cgroup-v1/cgroups.txt),
-namespace,
-[seccomp](https://www.kernel.org/doc/Documentation/prctl/seccomp_filter.txt)
-isolation and drops privileges of the Firecracker process.
```

Comment:
Maybe it'd be worth to keep this recommendation in a `## Firecracker` section.

Explicitly call out production workloads should use the default seccomp lvl 2

Reply:
done, please take another look